src/shared/ui/wizardPrompter.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { StepEstimator, Wizard } from '../wizards/wizard'
+import { Prompter, PromptResult } from './prompter'
+
+/**
+ * Wraps {@link Wizard} object into its own {@link Prompter}, allowing wizards to use other
```

Comment:
If Wizards can contain Wizards then we can probably merge the Prompter concept into the Wizard class. That eliminates the Prompter concept, so "Wizard" is the basic primitive.

Reply:
I'll see if I can get rid of this class. It's only being used in one place and it adds complexity that I would rather not have.